From 81d3e71027c8e702eba9961afa0220ae5e28066e Mon Sep 17 00:00:00 2001
From: Rich Megginson <rmeggins@redhat.com>
Date: Wed, 31 Jul 2013 09:53:52 -0600
Subject: [PATCH 97/99] fix compiler warning in posix winsync code for
 posix_group_del_memberuid_callback (cherry picked from commit
 f440e039a5f2a7b2ea0dd087d8e91c554abc1be0) (cherry picked from commit
 761d9d550ca1025293dc1607760adea38ddc5452) (cherry picked from commit
 9750ea73014b36349040e166a469ed09236c2549)

---
 ldap/servers/plugins/posix-winsync/posix-group-task.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/ldap/servers/plugins/posix-winsync/posix-group-task.c b/ldap/servers/plugins/posix-winsync/posix-group-task.c
index 1d9135d..05c43af 100644
--- a/ldap/servers/plugins/posix-winsync/posix-group-task.c
+++ b/ldap/servers/plugins/posix-winsync/posix-group-task.c
@@ -174,6 +174,7 @@ posix_group_task_destructor(Slapi_Task *task)
     }
 }
 
+#ifdef USE_POSIX_GROUP_DEL_MEMBERUID
 static int
 posix_group_del_memberuid_callback(Slapi_Entry *e, void *callback_data)
 {
@@ -208,6 +209,7 @@ posix_group_del_memberuid_callback(Slapi_Entry *e, void *callback_data)
 
     return rc;
 }
+#endif
 
 static int
 posix_group_fix_memberuid(char *dn, char *filter_str, void *txn)
-- 
1.8.1.4

